cli/flags.rs

```diff
@@ -1000,8 +1000,14 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
       Arg::with_name("fail-fast")
         .long("fail-fast")
         .alias("failfast")
-        .help("Stop on first error")
-        .takes_value(false),
+        .help("Stop after N errors")
+        .min_values(0)
+        .required(false)
+        .takes_value(true)
+        .validator(|val: String| match val.parse::<usize>() {
```

Comment:
```suggestion
        .value_name("N")
        .validator(|val: String| match val.parse::<usize>() {
```

Reply:
With these changes if we run 
```ts
./target/debug/deno test fetcher.ts --fail-fast 3
```
and it will throw 
```
error: Cannot resolve module "file:///home/getspooky/Documents/open-source/deno/3".
```

Reply:
Not with `require_equals`